data-prepper-expression/src/main/antlr/DataPrepperExpression.g4

```diff
@@ -225,27 +225,91 @@ EscapeSequence
     : '\\' [btnfr"'\\$]
     ;
 
-EQUAL : '==';
-NOT_EQUAL : '!=';
-LT : '<';
-GT : '>';
-LTE : '<=';
-GTE : '>=';
-MATCH_REGEX_PATTERN : '=~';
-NOT_MATCH_REGEX_PATTERN : '!~';
-IN_SET : 'in';
-NOT_IN_SET : 'not in';
-AND : 'and';
-OR : 'or';
-NOT : 'not';
-LPAREN : '(';
-RPAREN : ')';
-LBRACE : '{';
-RBRACE : '}';
-FORWARDSLASH : '/';
-DOUBLEQUOTE : '"';
-SET_SEPARATOR : ',';
+SET_DELIMITER
```

Comment:
Is there a particular reason to use the extra spacing here?

Reply:
I was expecting to make the lexer rules more complex when adding support for required space operators. I have reverted the extra spacing.